style.css

```diff
@@ -35,7 +35,79 @@ body {
 
 #navbar {
   margin-top: 13px;
-  margin-left: -30px;
+}
+
+/* styling for mobile menu transition */
+
+.nav-menu {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  gap: 60px;
+}
+
+.hamburger {
+  display: none;
+  cursor: pointer;
+}
+
+.bar {
+  display: block;
+  width: 25px;
+  height: 3px;
+  margin: 5px auto;
+  -webkit-transition: all 0.3s ease-in-out;
+  transition: all 0.3s ease-in-out;
+  background-color: black;
+}
+
+.nav-link {
+  font-family: "Poppins", sans-serif;
+  color: white;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 32px;
+  line-height: 44px;
+}
+
+@media (max-width: 768px) {
+  .hamburger {
+    display: block;
+  }
+
+  .hamburger.active .bar:nth-child(2) {
+    opacity: 0;
+  }
+
+  .hamburger.active .bar:nth-child(1) {
+    transform: translateY(8px) rotate(45deg);
+  }
+
+  .hamburger.active .bar:nth-child(3) {
+    transform: translateY(-8px) rotate(-45deg);
+  }
+
+  .nav-menu {
+    position: fixed;
+    left: -150%;
+    top: 70px;
+    gap: 0;
+    flex-direction: column;
+    background-color: #6070ff;
+    mix-blend-mode: multiply;
+    border-radius: 10px;
+    width: 80%;
+    text-align: center;
+    transition: 0.2s;
+  }
+
+  .nav-item {
+    margin: 18px 0;
+  }
+
+  .nav-menu.active {
+    left: 0;
+  }
```

Comment:
- [ ] The hamburger button and menu follow the design from the chosen template in Figma (e.g., font, colors, images, text, margins), please kindly fix your design 🙃 

Your design
<img width="372" alt="Screen Shot 2022-12-20 at 00 09 48" src="https://user-images.githubusercontent.com/74501523/208524792-44f706ad-8a72-4272-a23e-e44060cef045.png">

Figma design
<img width="327" alt="Screen Shot 2022-12-20 at 00 09 13" src="https://user-images.githubusercontent.com/74501523/208524953-fbdd8cdd-2412-4f2a-83a5-6bc5ee6474eb.png">

- [ ] Please kindly fix your mobile design. Your page should be responsive until the desktop screen size, you have some spaces on the right side of your page 🙃 🙃 

Your design
<img width="469" alt="Screen Shot 2022-12-20 at 00 10 58" src="https://user-images.githubusercontent.com/74501523/208525737-de1be5dd-ae56-49e3-9aed-b8d10cf1ffbe.png">

Figma design
<img width="300" alt="Screen Shot 2022-12-20 at 00 11 28" src="https://user-images.githubusercontent.com/74501523/208525866-734002f9-07ed-456d-aaa4-4dfaebf584e3.png">